CHANGELOG.md

```diff
@@ -7,6 +7,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v0.10.0]
+
+### Added
+
+- Implement gathering basic JVM metrics with micrometer.
```

Comment:
Personally, I'd rather these changelog entries were more end-user centric. So, rather than just copying all the PRs, instead explain to the user what the new feature is, and how they can get access to it.

Reply:
Good point, done. I'll also update the release notes so that they're a bit more user-friendly.